src/shapes/path.class.js

```diff
@@ -335,20 +335,10 @@
    * @static
    * @memberOf fabric.Path
    * @param {Object} object
-   * @param {Function} [callback] Callback to invoke when an fabric.Path instance is created
+   * @returns {Promise<fabric.Path>}
    */
-  fabric.Path.fromObject = function(object, callback) {
-    if (typeof object.sourcePath === 'string') {
-      var pathUrl = object.sourcePath;
-      fabric.loadSVGFromURL(pathUrl, function (elements) {
```

Comment:
also this feature, away.
This is a reminescensce of pathGroup, in which if you exported to a data-less object, the path would be converted in a path-string or an svg.